queries/leo/highlights.scm

```diff
@@ -0,0 +1,264 @@
+((identifier) @variable (#set! "priority" 95))
```

Comment:
can you explain the reason why you use priority here

Reply:
No! :) deleted [01fc7f6359a](https://github.com/r001/nvim-treesitter/commit/01fc7f6359ab7d708a00bdfec10e7fa014cf0fdf)